doc/source/serve/deployment-graph-cookbook.md

```diff
@@ -0,0 +1,60 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.13.6
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(serve-deployment-graph-cookbook)=
+
+# Deployment Graph Cookbook
+
+### Chain Nodes
+
+The example shows how to chain nodes using the same class and having different args passed in
+
++++
+
+```python
+import ray
+from ray import serve
+from ray.experimental.dag.input_node import InputNode
+from ray.serve.drivers import DAGDriver
+
+ray.init(num_cpus=16)
```

Comment:
we can simplify this example by lowering # of nodes used since not every user's laptop has 16 cores (ex: M1 laptop) for this.

Reply:
lowered the number and add comments in the code